composer/src/lib.rs

```diff
@@ -289,21 +335,56 @@ impl Builder {
         self
     }
 
+    /// prune containers and networks on start
     pub fn with_prune(mut self, enable: bool) -> Builder {
         self.prune = enable;
         self
     }
+
+    /// output logs on panic
+    pub fn with_logs(mut self, enable: bool) -> Builder {
+        self.logs_on_panic = enable;
+        self
+    }
+
+    /// setup tracing for the cargo test code with `RUST_LOG` const
+    pub fn with_default_tracing(self) -> Self {
+        if let Ok(filter) =
+            tracing_subscriber::EnvFilter::try_from_default_env()
+        {
+            tracing_subscriber::fmt().with_env_filter(filter).init();
+        } else {
+            tracing_subscriber::fmt().with_env_filter(RUST_LOG).init();
+        }
+        self
+    }
+
+    /// setup tracing for the cargo test code with `filter`
+    pub fn with_tracing(self, filter: &str) -> Self {
+        if let Ok(filter) =
+            tracing_subscriber::EnvFilter::try_from_default_env()
+        {
```

Comment:
These two cases seem to have the same code

Reply:
oh yeah, I got one to call the other now, thanks